website_wow/static/src/js/website_wow.js

```diff
@@ -0,0 +1,39 @@
+/* Copyright 2017 LasLabs Inc.
+   License GPL-3.0 or later (http://www.gnu.org/licenses/gpl).
+*/
+
+odoo.define('website_wow.website_wow', function (require) {
+    'use strict';
+
+    var base = require('web_editor.base');
+    var core = require('web.core');
+
+    var WebsiteWow = {
```

Comment:
Could this extend `web.Class`, so we have the common Odoo JS framework boilerplate? Or even better from `web_editor.snippets.animation.Class`?